cfy_manager/status_reporter/status_reporter.py

```diff
@@ -13,18 +13,96 @@
 #  * See the License for the specific language governing permissions and
 #  * limitations under the License.
 
+import json
+
+import argh
+import yaml
+
 from ..utils.systemd import systemd
-from ..constants import STATUS_REPORTER
+from ..utils.files import update_yaml_file, sudo_read
 from ..logger import get_logger, setup_console_logger
+from ..constants import STATUS_REPORTER, STATUS_REPORTER_CONFIGURATION_PATH
 
 logger = get_logger(STATUS_REPORTER)
 setup_console_logger()
 
 
-def configure():
+@argh.arg('--managers-ip',
+          nargs='+',
+          type=str,
+          help='Cloudify managers ip list status reporter will report to, '
+               'Example: `<ip-1> <ip-2> <ip-3>`'
+          )
+@argh.arg('--user-name',
+          type=str,
+          help='The status reporter\'s user name in the Cloudify system, '
+               'Example: `manager_reporter`.'
+          )
+@argh.arg('--token',
+          type=str,
+          help='The login auth token for the status reporter\'s user'
+               ' in the Cloudify system.'
+          )
+@argh.arg('--ca-path',
+          type=str,
+          help='A local path to the CA certificate for communication with'
+               ' the Cloudify managers.'
+          )
+@argh.arg('--reporting-freq',
+          type=int,
```

Comment:
Don't we have defaults? for some of the arguments

Reply:
at the file level we have defaulted that, we could maybe add some more